sites/fast-website/src/app/components/card-section/card-section.styles.ts

```diff
@@ -1,12 +1,13 @@
 import { css } from "@microsoft/fast-element";
 import { display } from "@microsoft/fast-foundation";
+import { siteBreakpoints } from "../../style/constants";
```

Comment:
We can add an alias to `webpack.common.js` so we can easily reference this module. In `module.exports.resolve.alias`, add a line similar to the `svg` property. This will allow us to reference `style/constants` directly without having to do manual path finding.

Then, these can be imported with proper their names:

```ts
import {
    normalDeviceWidth, // see my other comment on constants.ts
    bigDeviceWidth
} from "style/constants";
```

Reply:
That sounds like a great idea.  I haven't done much work directly in webpack and that was a while ago, but I'm game to give it a go.

Reply:
You will also need to tell *TypeScript* about this alias. Historically we've avoided this as it seemed to cause more issues than it really solved, but if you think its helpful go for it - just note the above.

Reply:
Reverting this to fixed numbers for now.  Proposal coming forth on this.  See Chris' comment below.